contracts/package.json

```diff
@@ -75,30 +71,30 @@
     "deep-equal-in-any-order": "^2.0.6",
     "eslint": "^8.57.1",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.2.3",
-    "ethers": "~5.7.2",
+    "eslint-plugin-prettier": "^5.2.5",
+    "ethers": "~5.8.0",
     "hardhat": "~2.20.1",
     "hardhat-abi-exporter": "^2.10.1",
     "hardhat-ignore-warnings": "^0.2.12",
     "moment": "^2.30.1",
-    "prettier": "^3.5.0",
+    "prettier": "^3.5.3",
     "prettier-plugin-solidity": "^1.4.2",
     "solhint": "^5.0.5",
-    "solhint-plugin-chainlink-solidity": "git+https://github.com/smartcontractkit/chainlink-solhint-rules.git#v1.2.1",
+    "solhint-plugin-chainlink-solidity": "github:smartcontractkit/chainlink-solhint-rules#v1.2.1",
     "solhint-plugin-prettier": "^0.1.0",
     "ts-node": "^10.9.2",
-    "typechain": "^8.2.1",
-    "typescript": "^5.7.3"
+    "typechain": "^8.3.2",
+    "typescript": "^5.8.2"
   },
   "dependencies": {
-    "@arbitrum/nitro-contracts": "1.1.1",
```

Comment:
We don't use any of the files that are changed between 1.1.1 and 3.0.0 so this upgrade is safe